website/pages/docs/concepts/integrated-storage.mdx

```diff
@@ -0,0 +1,141 @@
+---
+layout: docs
+page_title: Integrated Storage
+sidebar_title: Integrated Storage
+description: Learn about the integrated raft storage in Vault.
+---
+
+# Integrated Storage
+
+Vault supports a number of Storage options for the durable storage of Vault's
+information. As of Vault 1.4 an integrated storage option is offered. This
+storage backend does not rely on any third party systems, it implements high
+availability, supports Enterprise Replication features, and provides
+backup/restore work flows.
+
+The integrated storage option stores Vault's data on the server's filesystem and
+uses a consensus protocol to replicate data to each server in the cluster. More
+information on the internals of integrated storage can be found in the
+[integrated storage internals
+documentation](/docs/internals/integrated-storage/). Also the 
+[Configuration](/docs/configuration/storage/raft/) docs can help in configuring
+Vault to used integrated storage.
+
+The sections below go into various details on how to operate Vault with
+integrated storage.
+
+## Cluster Membership
+
+This section will outline how to bootstrap and manage a cluster of Vault nodes
+running integrated storage. 
+
+Integrated storage is bootstrapped during the [initialization
+process](https://learn.hashicorp.com/vault/getting-started/deploy#initializing-the-vault),
+and results in a cluster of size 1. Depending on the [desired deployment
+size](/docs/internals/integrated-storage/#deployment-table), nodes can be joined
+to the active Vault node.
+
+### Joining Nodes
+
+Joining is the process of taking an uninitialized Vault node and making it a
+member of an existing cluster. In order to authenticate the new node to the
+cluster it must use the same seal mechanism. If using a Auto Unseal the node
+must be configured to use the same KMS provider and Key as the cluster it's
+attempting to join. If using a Shamir seal the unseal keys must be provided to
+the new node before the join process can complete. Once a node has successfully
+joined data from the active node can begin to replicate to it. Once a node has
+been joined it cannot be re-joined to a different cluster.
+
+You can either join the node automatically via the config file or manually through the
```

Comment:
Link to the respective config here would be helpful.

Reply:
I thought that the first time I read it too, but the configs are described right below this, so I wasn't sure how useful it would be.